prisondersdilemma.py

```diff
@@ -11,182 +11,260 @@
 from numpy.random import shuffle
 import numpy as np
 import random
+from win32api import GetSystemMetrics
 
 #these are all just building blocks for now, not functioning code! it won't run yet
 
 #create window
 
-DISPSIZE = (1000, 700)
+DISPSIZE = (GetSystemMetrics(0),GetSystemMetrics(1))
 BGC = 'white'
 FGC = (-1,-1,-1) #defining colour of text shown => black
+#log file
+log = open ('logfileprisonersdilemma.txt', 'w')
+log.write(' .... ') #insert headers of what should be logged
 
 win = Window(size=DISPSIZE, units='pix', color = BGC, fullscr=False)
```

Comment:
A minor point but the code would look better if you were consistent with the spacing (e.g., color = BGC, fullscr=False).